### DISCUSSION TITLE
How to make scrolling smoother

### DISCUSSION THREAD

**Discussion by Yutsano0** [sumatrapdfreader/sumatrapdf#2634] [Q&A] [answered]
been using sumatra for a month now and still i don't know how to make scrolling smoother like the scrolling is jittery and its kinda hurts in my eyes.

**Comment by kjk** [comment, marked as answer]
In 3.4 try experimental smooth scrolling: https://www.sumatrapdfreader.org/settings/settings#SmoothScroll, https://www.sumatrapdfreader.org/docs/Advanced-options-settings

**Reply by Yutsano0** [reply]
its not smooth like other pdf viewers but it works.
